source/generators/LED450LW.cc

```diff
@@ -0,0 +1,151 @@
+// ----------------------------------------------------------------------------
+// nexus | LED450LW.cc
+//
+// This class is the primary generator for events consisting of
```

Comment:
Is this class a copy of `SingleParticleGenerator` or is there any difference that I'm not spotting?